cmd/spiffe-aws-assume-role/cli/iso8601DurationMapper_test.go

```diff
@@ -0,0 +1,46 @@
+package cli
+
+import (
+	"github.com/alecthomas/kong"
+	"github.com/stretchr/testify/require"
+	"reflect"
+	"testing"
+	"time"
+)
+
+func TestStandardLibraryCannotParseIso8601Duration(t *testing.T) {
+	_, err := time.ParseDuration("PT5M")
+	require.Error(t, err, "Standard library was able to parse ISO8601 duration string, iso8601DurationMapper may no longer be needed")
+}
```

Comment:
Wouldn't that be nice?

Reply:
Oh, that's a terrific idea to have it as a test!

Reply:
I'm a big fan of tests-as-documentation, even if it's just to verify some built-in platform/SDK behavior. Someone might come across the ISO 8601 code later and scratch their head as to why we're bothering to do it all by hand, I think the test makes it pretty clear.